src/material/schematics/ng-generate/navigation/files/__path__/__name@dasherize@if-flat__/__name@dasherize__.component.html.template

```diff
@@ -5,9 +5,9 @@
       [opened]="(isHandset$ | async) === false">
     <mat-toolbar>Menu</mat-toolbar>
     <mat-nav-list>
-      <a mat-list-item href="#">Link 1</a>
-      <a mat-list-item href="#">Link 2</a>
-      <a mat-list-item href="#">Link 3</a>
+      <a mat-list-item <%= routing === true ? 'routerLink="/"' : 'href="#"' %>>Link 1</a>
```

Comment:
You should be able to do just `routing ? 'routerLink="/"' : 'href="#"'`.

Reply:
@crisbeto done.